manifests/c/Cloudflare/cloudflared/2022.6.2/Cloudflare.cloudflared.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Cloudflare.cloudflared
+PackageVersion: 2022.6.2
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+ReleaseDate: 2022-06-17
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/cloudflare/cloudflared/releases/download/2022.6.2/cloudflared-windows-amd64.msi
+  InstallerSha256: BDFCBD9D8DBB3D5F3A8C08A9176A19BD2A19E3740CFC7815E87A7E103A202314
```

```suggestion
  InstallerSha256: BDFCBD9D8DBB3D5F3A8C08A9176A19BD2A19E3740CFC7815E87A7E103A202314
  ProductCode: "{35E5E858-9372-4449-BF73-1CD6F7267128}"
```
`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{35E5E858-9372-4449-BF73-1CD6F7267128}`